nixos/modules/services/desktop-managers/plasma6.nix

```diff
@@ -245,6 +245,9 @@ in {
     xdg.portal.configPackages = mkDefault [kdePackages.xdg-desktop-portal-kde];
     services.pipewire.enable = mkDefault true;
 
+    # Enable screen reader by default
+    services.orca.enable = mkDefault true;
```

Comment:
I don't think we should turn on all the optional components by default. The default installation and ISO will just grow out of hand.

Reply:
It's for accessibility. ISO size is better than people not being able to use the system.

Reply:
> ISO size is better than people not being able to use the system.

IMO if you live in a country with embarrassingly slow internet aka Germany I'd rather take the 1.8GB Plasma 5 ISO than the current 3.2GB Plasma 6 one.

IMO we should probably do a more minimal GUI installer and a complete one which maybe is only there for stable to save some storage.

> KDE also explicitly say that distros should include Orca

That also mentions the following things we don't do:
- enable iio-sensor-proxy
- install icoutils 
- we install maliit only in the ISO, not a default plasma installation https://github.com/NixOS/nixpkgs/blob/7fa0c16b94c39c3a3462ca710c968a4617349155/nixos/modules/installer/cd-dvd/installation-cd-graphical-calamares-plasma6.nix#L23-L24
- enable switcheroo-control
- we don't enable kdeconnect by default or open its ports in the firewall
- I don't think we enable module-x11-bell by default (didn't double check)
- or set the recommended firefox configuration, not even in programs.firefox

Reply:
> IMO if you live in a country with embarrassingly slow internet aka Germany I'd rather take the 1.8GB Plasma 5 ISO than the current 3.2GB Plasma 6 one.

I doubt this has much to do with Orca.

I also don’t think that telling blind people they can only install stable is sensible.